mesh/mesh_test.go

```diff
@@ -276,91 +274,3 @@ func TestLayers_OrphanBlocks(t *testing.T) {
 	assert.True(t, len(arr3) == 1, "wrong layer")
 
 }
-
-func createLayerWithAtx(t *testing.T, msh *Mesh, id types.LayerID, numOfBlocks int, atxs []*types.ActivationTx, votes []types.BlockID, views []types.BlockID, context bool) (created []types.BlockID) {
-	if numOfBlocks < len(atxs) {
-		panic("not supported")
-	}
-	for i := 0; i < numOfBlocks; i++ {
-		rid := rand.Uint64()
-		bid := types.BlockID(rid)
-		block1 := types.NewExistingBlock(bid, id, []byte("data1"))
-		block1.BlockVotes = append(block1.BlockVotes, votes...)
-		if i < len(atxs) {
-			block1.AtxIds = append(block1.AtxIds, atxs[i].Id())
-			fmt.Printf("adding i=%v bid=%v atxid=%v", i, bid, atxs[i].Id().String())
-		}
-		block1.ViewEdges = append(block1.ViewEdges, views...)
-		msh.setContextualValidity(bid, context)
-		var actualAtxs []*types.ActivationTx
-		if i < len(atxs) {
-			actualAtxs = atxs[i : i+1]
-		}
-		err := msh.AddBlockWithTxs(block1, []*types.AddressableSignedTransaction{}, actualAtxs)
-		require.NoError(t, err)
-		created = append(created, block1.Id)
-	}
-	return
-}
-
-func rndStr() string {
-	a := make([]byte, 4)
-	rand.Read(a)
-	return string(a)
-}
-
-func TestMesh_ActiveSetForLayerView(t *testing.T) {
```

Comment:
we need to tests this code, make sure you have these tests somewhere else

Reply:
activation_test